Tests/lib/vsts-task-lib/task.ts

```diff
@@ -666,9 +666,12 @@ export function matchFile(list, pattern, options): string[] {
     return mock.getResponse('match', pattern) || [];
 }
 
-export function filter(pattern, options): string[] {
-    return mock.getResponse('filter', pattern) || [];
-}    
+export function filter(pattern, options): any {
+	var filterList = mock.getResponse('filter', pattern) || [];
+	return function(pattern, i, filterList) {
+		return filterList;
```

Comment:
If you use .filter, then should your answers be an array of booleans. It's confusing. Would it be simpler to use tl.match instead?